File: citicup-admin/schema/market.go
package schema

//行情数据

//市场行情		TODO:这里的市场行情是一天的？
type Market struct {
	ID       uint    `json:"id"`
	Tscode   string  `json:"column:ts_code"` //股票代码
	TradeDate string  `json:"trade_date"` //交易日期
	Open     float32 `json:"open"`           //开盘价
	Amount   float32 `json:"amount"`         //成交额（亿）
	Change   float32 `json:"change"`         //涨跌额
	PctChg   float32 `json:"pct_chg"`        //涨跌幅
	Close    float32 `json:"close"`          //收盘价
	Highest  float32 `json:"high"`           //最高价
	Lowest   float32 `json:"low"`            //最低价
	PreClose float32 `json:"pre_close"`      //昨日收盘价
	Vol      float32 `json:"vol"`            //成交量
}

type InsertMarketParam struct {
	Market    Market `json:"market"`
	CompanyID uint   `json:"company_id"`
}

File: citicup-plus/src/main/java/com/example/demo/model/Market.java
package com.example.demo.model;

import javax.persistence.Entity;
import javax.persistence.Table;

//日线行情
@Entity
@Table(name = "market")
public class Market extends BaseEntity {
    private String tsCode;
    private String tradeDate;
    private double open;
    private double amount;
    private double change;
    private double pctChg;
    private double close;
    private double high;
    private double low;
    private double preClose;
    private double vol;

    public String getTsCode() {
        return tsCode;
    }

    public void setTsCode(String tsCode) {
        this.tsCode = tsCode;
    }

    public String getTradeDate() {
        return tradeDate;
    }

    public void setTradeDate(String tradeDate) {
        this.tradeDate = tradeDate;
    }

    public double getOpen() {
        return open;
    }

    public void setOpen(double open) {
        this.open = open;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public double getChange() {
        return change;
    }

    public void setChange(double change) {
        this.change = change;
    }

    public double getPctChg() {
        return pctChg;
    }

    public void setPctChg(double pctChg) {
        this.pctChg = pctChg;
    }

    public double getClose() {
        return close;
    }

    public void setClose(double close) {
        this.close = close;
    }

    public double getHigh() {
        return high;
    }

    public void setHigh(double high) {
        this.high = high;
    }

    public double getLow() {
        return low;
    }

    public void setLow(double low) {
        this.low = low;
    }

    public double getPreClose() {
        return preClose;
    }

    public void setPreClose(double preClose) {
        this.preClose = preClose;
    }

    public double getVol() {
        return vol;
    }

    public void setVol(double vol) {
        this.vol = vol;
    }
}
